src/test/resources/json_settings/settings.json

```diff
@@ -0,0 +1,49 @@
+[{
```

Comment:
Can this file be deleted?

Reply:
Removed it. It was supposed to be sample for testing